### PR TITLE
Implements sqlite plugin

### PR DESCRIPTION
**Pull request by javanlacerda**

### CODE REVIEW

File: broker/persistence/sqlite/model.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# -*- coding: utf_8 -*-
+import peewee
+
+db = peewee.SqliteDatabase('./local_database/sqlite.db')
```
Comment (clenimar) [inline broker/persistence/sqlite/model.py]: this path should be customizable through a config option in the appropriate section.

File: broker/persistence/sqlite/plugin.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+import dill
+import peewee
+
+
+class SqlitePersistence(PersistenceInterface):
+
+    def __init__(self):
+        try:
+            KubeJobsState.create_table()
```
Comment (clenimar) [inline broker/persistence/sqlite/plugin.py]: this implementation seems to be highly tied to KubeJobs plugin. shouldn't it be more generic and plugin-agnostic?

File: broker/tests/unit/mocks/k8s_mock.py [hunk 3]
```diff
@@ -173,3 +173,15 @@ def create_influxdb(app_id, database_name="asperathos",
...
         return {"port": 1234, "name": database_name}
+
+    """ Function that simulates a request to job status.
+
+    Args:
+        app_id (string): Representing id of the application
+
+    Returns:
+        None
+    """
```
Comment (clenimar) [inline broker/tests/unit/mocks/k8s_mock.py]: move the docstring to method's body.

File: broker/tests/unit/mocks/k8s_mock.py [hunk 4]
```diff
@@ -173,3 +173,15 @@ def create_influxdb(app_id, database_name="asperathos",
...
+    def get_job_status(app_id, namespace="default"):
```
Comment (clenimar) [inline broker/tests/unit/mocks/k8s_mock.py]: you don't seem to use the parameter `namespace`.

**Comment by matheuscmelo** [comment]
Build is failing because of new config parameters. Please make a PR to https://github.com/ufcg-lsd/asperathos-integration-tests updating broker.cfg or change your code so it doesn't rely on config files.

**Pull request merged**